goalie/field.py

```diff
@@ -27,10 +27,11 @@ def __init__(
         The finite element for the Field should be set either using the `finite_element`
         keyword argument or a combination of the `mesh`, `family`, `degree`, `vfamily`,
         `vdegree`, and/or `variant` keyword arguments. For details on these arguments,
-        see :class:`firedrake.functionspace.FunctionSpace`.
-
-        If neither are specified, the default finite element is a scalar Real space on
-        and interval.
+        see :class:`firedrake.functionspace.FunctionSpace`. 
+        
+        If the `finite_element` keyword argument is specified, these other arguments are
+        ignored. If neither are specified, the default finite element is a scalar Real
+        space on an interval.
```

Comment:
```suggestion
        If the `finite_element` keyword argument is specified, these other arguments are
        ignored. If neither are specified, the default finite element is a scalar Real
        space.
```
There's no longer a concept of cell at this point, right?

Reply:
Thanks! Committed the suggestion https://github.com/mesh-adaptation/goalie/pull/301/commits/1a84692a086e21325286c953ac83009a8274d847.